receiver/mongodbatlasreceiver/alerts.go

```diff
@@ -204,6 +327,86 @@ func (a alertsReceiver) Shutdown(ctx context.Context) error {
 	return nil
 }
 
+func (a alertsReceiver) shutdownPoller(ctx context.Context) error {
+	a.logger.Debug("Shutting down client")
+	close(a.doneChan)
+	a.wg.Wait()
+	return a.writeCheckpoint(ctx)
+}
+
+func (a alertsReceiver) convertAlerts(now pcommon.Timestamp, alerts []mongodbatlas.Alert) (plog.Logs, error) {
+	logs := plog.NewLogs()
+	var errs error
+	for _, alert := range alerts {
+		resourceLogs := logs.ResourceLogs().AppendEmpty()
+		logRecord := resourceLogs.ScopeLogs().AppendEmpty().LogRecords().AppendEmpty()
+		logRecord.SetObservedTimestamp(now)
+
+		ts, err := time.Parse(time.RFC3339, alert.Updated)
+		if err != nil {
+			a.logger.Warn("unable to interpret updated time for alert, expecting a RFC3339 timestamp", zap.String("timestamp", alert.Updated))
+			continue
+		}
+
+		logRecord.SetTimestamp(pcommon.NewTimestampFromTime(ts))
+		logRecord.SetSeverityNumber(severityFromAPIAlert(alert))
```

Comment:
We should set the severity text as well. It should be the original value (eg "OPEN").

Reply:
set to report the severity text as the status 👍

Reply:
Related nit: just pass in alert.Status to severity mapping function too.